docs/tutorials/using-pixy-camera.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Using the CMUcam5 Pixy for LEGO Mindstorms
+group: hardware-extras
+author: "Kees Smit"
+---
+
+* Table of Contents
+{:toc}
+
+## Intro
+
+The CMUcam5 Pixy camera is available in a version designed for
+LEGO MIndstorms. In this tutorial we explain how to use it with ev3dev
+and python.
+General information about Pixy for LEGO MIndstorms can be found on
+Charmed Lab's [wiki-page](http://cmucam.org/projects/cmucam5/wiki/LEGO_Wiki).
+
+## What you need
+
+* LEGO MIndstorms EV3.
+* CMUcam5 Pixy for LEGO Mindstorms, connected to the EV3 with the cable
+deliverd with your camera. If you have a regular version of CMUcam5 Pixy
+you will need a special Pixy adapter for EV3.
+
+**Note:** the Pixy camera comes with it's own tool: PixyMon. This tool helps
+you to set the signatures (object you want Pixy to detect). To use this tool
+you need to connect the camera directly to your PC by using a mini USB cable.
+For this you can use the USB cable of your EV3. Beware that when the camera 
+is connected to your PC this way, it's valuesare not updated! So before 
+starting your script be sure the camerais disconnected form your computer 
+and thus only connected to your EV3.
+
+## Connecting to the camera and reading data
+
+Use the `Sensor` class to connect to the Pixy camera:
+
+    pixy = Sensor(address='in1')
```

Comment:
Also, you can get python syntax highlighting by writing this as 

    ```python
    pixy = Sensor(address='in1')
    assert pixy.connected, "Error while connecting Pixy camera to port1"
    ```

instead of indenting 4 spaces